atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -85,6 +85,8 @@ protected final void check() {
 
     @Value.Derived
     public LockLeader whoIsTheLockLeader() {
-        return LockLeader.iAmTheLockLeader(lockCreator().equals(localServer()));
```

Comment:
Switched to `fromBoolean`.

I see what you mean, but I don't really agree with that. I try and make enums a Plain Old Data Structure. Also, something like `fromBoolean` I don't count as logic since all of the (important) logic of which one is going to be selected is kept in `whoIsTheLockLeader` anyway.